build/PlatformInstaller.targets

```diff
@@ -0,0 +1,298 @@
+
+    <!-- Create layout: Extract archive if given -->
+    <MakeDir Directories="$(_WorkRoot)package_root\" />
+    <Exec Command="tar -xzf $(SFArchive) -C $(_WorkRoot)package_root\" Condition="'$(SFArchive)'!=''" />
+
+    <!-- Create layout: Place changelog -->
+    <Copy
+      SourceFiles="$(_PackagingDir)changelog"
+      DestinationFiles="$(_WorkRoot)templates/changelog"
+      OverwriteReadOnlyFiles="True"
+      SkipUnchangedFiles="False"
+      UseHardlinksIfPossible="False" />
+
+    <ItemGroup>
+      <ChangelogItems Include="DATE" Replacement="$([System.DateTime]::UtcNow.ToString(ddd MMM dd yyyy))" />
+      <ChangelogItems Include="MAINTAINER_NAME" Replacement="$(RPMMaintainerName)" />
+      <ChangelogItems Include="MAINTAINER_EMAIL" Replacement="$(RPMMaintainerEmail)" />
+      <ChangelogItems Include="PACKAGE_VERSION" Replacement="$(RPMVersion)" />
+      <ChangelogItems Include="PACKAGE_REVISION" Replacement="$(RPMRevision)" />
+    </ItemGroup>
+
+    <!-- Update Date, Maintainer, Version, Revision and Changelog Message -->
+    <RepoTasks.ReplaceInFile Filename="$(_WorkRoot)templates/changelog" Items="@(ChangelogItems)" />
+
+    <!-- Update Maintainer and Summary -->
+    <PropertyGroup>
+      <RPMMaintainer>$(RPMMaintainerName) &lt;$(RPMMaintainerEmail)&gt;</RPMMaintainer>
+      <RPMSummary>$(RPMSummary.Replace('DEB_VERSION','$(RPMVersion)'))</RPMSummary>
+    </PropertyGroup>
+
+    <!-- Run RPM -->
+    <Exec Command="docker run
+      --rm
+      -v $(RepositoryRoot):$(_DockerRootDir)
+      docker-image-$(Image)
+      fpm
+        --verbose
+        -s dir
+        -t rpm
+        -n $(RPMInstallerPrefix)-$(RPMVersion)
+        -p $(_DockerRootDir)artifacts/installers/$(RPMInstallerPrefix)-$(RPMVersion)-$(RPMFileSuffix)
+        -v $(RPMVersion)
+        --iteration $(RPMRevision)
+        -a amd64
+        $(RPMArguments)
+        --rpm-changelog $(_DockerRootDir).w/templates/changelog
+        --rpm-summary &quot;$(RPMSummary)&quot;
+        --description &quot;$(RPMDescription)&quot;
+        --maintainer &quot;$(RPMMaintainer)&quot;
+        --vendor &quot;$(RPMVendor)&quot;
+        --license &quot;$(RPMLicense)&quot;
+        --url &quot;$(RPMHomepage)&quot;
+        $(_DockerRootDir).w/package_root/=&quot;$(RPMInstallRoot)/&quot;" />
+  </Target>
+
+  <Target Name="GenerateRpms" DependsOnTargets="_EnsureInstallerPrerequisites">
+     <PropertyGroup>
+      <Image>rhel.7</Image>
+      <RPMVendor>.NET Foundation</RPMVendor>
+      <RHInstallRoot>/opt/rh/rh-dotnet20/root/usr/lib64/dotnet</RHInstallRoot>
+    </PropertyGroup>
+
+    <!-- Build Docker Image -->
+    <MSBuild Projects="$(MSBuildProjectFullPath)" Targets="_BuildDockerImage" Properties="Image=$(Image)" />
+
+    <ItemGroup>
+      <RPMHostingDependencies Include="$(SFInstallerName)-$(PackageVersion)" Version="$(PackageVersion)" />
+      <RPMHostingDependencies Include="$(DotnetRuntimeInstallerName)" Version="$(CoreSetupPackageVersion)" />
```

Comment:
TODO: aspnetcore-shared installer needs to take a dependency on the previous 2.0.3 aspnetcore-store installer until "trampoline" is implemented for 2.1 preview2

Reply:
Add this as a comment to the code and add a work item to remove it for 2.1 RTM.